docker-micro-benchmark/helpers/docker_helpers.go

```diff
@@ -244,9 +285,9 @@ func DoParallelContainerStopBenchmark(client *docker.Client, qps float64, routin
 
 // GetContainerIDs returns all the container ids in the system
 func GetContainerIDs(client *docker.Client) (containerIDs []string) {
-	containers, err := client.ListContainers(docker.ListContainersOptions{All: true})
+	containers, err := client.ContainerList(getContext(), types.ContainerListOptions{All: true})
 	if err != nil {
-		panic(fmt.Sprintf("Error list containers: %v", err))
+		LogError("Error list containers: %v", err)
```

Comment:
I see you added a lot of panics  in the PR. I'm concerned that the benchmark will never complete given that it panics whenever there is any error. How about we do something simpler:
 - For operations that are necessary for the test to start, panic or error out and stop the tests (and retry if possible).
 - For others, record the error (e.g., in a counter) and move on. Report the overall error at the end for the human beings to judge whether this is acceptable.  

This should be simple enough and does not require too much work. Let me know if you disagree.